senza/traffic.py

```diff
@@ -395,3 +396,12 @@ def inform_sns(arns: list, message: str, region):
     sns = boto3.client('sns', region_name=region)
     for sns_topic in sns_topics:
         sns.publish(TopicArn=sns_topic, Subject="SenzaTrafficRedirect", Message=jsonizer.encode((message)))
+
+
+def resolve_to_ip_addresses(dns_name: str) -> set:
+    try:
+        answers = dns.resolver.query(dns_name, 'A')
+    except:
```

Comment:
@rafaelcaricio I personally don't care if the contents of the `try` clause only have one call to a 3rd party library --- I don't want to look up what kind of exceptions the 3rd party raises, I just want to ignore any error..